CMGTools/TTHAnalysis/cfg/crab/heppy_crab_config_env.py

```diff
@@ -2,47 +2,42 @@
 # values we'll be taken from environment variables set in launchall.py
 # fixed options will be taken from heppy_crab_config.py
 
-import imp
+import imp, os
 file = open( "heppy_crab_config.py", 'r' )
 cfg = imp.load_source( 'cfg', "heppy_crab_config.py", file)
 config = cfg.config
-import os
-import re
-dataset=os.environ["DATASET"]
-NJOBS=int(os.environ["NJOBS"])
-production_label = os.environ["PROD_LABEL"]
-cmg_version = os.environ["CMG_VERSION"]
-debug  = os.environ["DEBUG"] == 'True'
-useAAA = os.environ["USEAAA"] == 'True'
 
-if debug:
-    NJOBS = 4
-    NEVENTS = 200
+print "Will send dataset", os.environ["DATASET"], "with", os.environ["NJOBS"], "jobs"
 
-print "Will send dataset", dataset, "with", NJOBS, "jobs"
-
-config.General.requestName = dataset + "_" + cmg_version # task name
-config.General.workArea = 'crab_' + production_label # crab dir name
+config.General.requestName = os.environ["DATASET"] + "_" + os.environ["CMG_VERSION"] # task name
+config.General.workArea = 'crab_' + os.environ["DATASET"] # crab dir name
```

Comment:
Ciao Marco, all this is pretty cool, thanks!

just one little comment on the line above. If one repeats one dataset in different production, the crab working directory will be the same, no?. Why not keep crab_production_label as before for a cleaner bookkeeping?